src/brogue/Recordings.c

```diff
@@ -65,17 +65,17 @@ unsigned char compressKeystroke(long c) {
     return UNKNOWN_KEY;
 }
 
-void numberToString(unsigned long number, short numberOfBytes, unsigned char *recordTo) {
+void numberToString(uint64_t number, short numberOfBytes, unsigned char *recordTo) {
     short i;
-    unsigned long n;
+    uint64_t n;
 
     n = number;
     for (i=numberOfBytes - 1; i >= 0; i--) {
         recordTo[i] = n % 256;
         n /= 256;
     }
     if (n > 0) {
-        printf("\nError: the number %li does not fit in %i bytes.", number, numberOfBytes);
+        printf("\nError: the number %llu does not fit in %i bytes.", number, numberOfBytes);
         brogueAssert(false);
     }
```

Comment:
Was this branch triggering with the old bug?

Reply:
I'm not sure actually. This caught my eye when I looked for usages of the uint64_t data type. I found a stack overflow article that suggests using %llu, so I made the change.